client/src/components/DistressGraphs/AppetiteGraph.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
```

Comment:
Another advice for all these graphs. Instead of creating all these different graphs for each specific X/Y axis, can we instead create a graph template that allows you to fill in things like "title, yaxis name, yaxis intervals, etc"

That way we don't need to copy/paste each graph and can instead everytime we have a graph object, we can just call the template instead?

Reply:
That is smart I will look into doing this, thanks.